gitops/environments/base/deployment.yaml

```diff
@@ -2,37 +2,31 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: "<WL_SERVICE_NAME>"
-  annotations: {
-    reloader.stakater.com/auto: "true",
-    # <K8S_ROLE_MAPPING>: "<WL_IAM_ROLE_RN>"
-  }
-  labels: {
-    # <ADDITIONAL_LABELS>
```

Comment:
This annotation is extremely important, and this snippet will be replaced during parametrization. Please return it back.